ui/src/app/shared/pipes/dataflow-date-time.pipe.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { DataflowDateTimePipe } from './dataflow-date-time.pipe';
+import * as moment from 'moment';
+
+describe('DataflowDateTimePipe', () => {
+
+  let pipe: DataflowDateTimePipe;
+
+  beforeEach(() => {
+    pipe = new DataflowDateTimePipe();
+  });
+
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('convert string', () => {
+    expect(pipe.transform('1970')).toEqual('1970-01-01T00:00:00.000Z');
```

Comment:
Test fails in timezone out side of GMT.  Exception I'm getting is :

```
	Expected '1969-12-31T19:00:00.000Z' to equal '1970-01-01T00:00:00.000Z'.
	    at UserContext.<anonymous> Users/glennrenfro/project/spring-cloud-dataflow-ui/ui/src/app/shared/pipes/dataflow-date-time.pipe.spec.ts:17:27)
	    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke Users/glennrenfro/project/spring-cloud-dataflow-ui/ui/node_modules/zone.js/dist/zone.js:391:1)
	    at ProxyZoneSpec.webpackJsonp.../../../../zone.js/dist/proxy.js.ProxyZoneSpec.onInvoke Users/glennrenfro/project/spring-cloud-dataflow-ui/ui/node_modules/zone.js/dist/proxy.js:79:1)
	    at ZoneDelegate.webpackJsonp.../../../../zone.js/dist/zone.js.ZoneDelegate.invoke Users/glennrenfro/project/spring-cloud-dataflow-ui/ui/node_modules/zone.js/dist/zone.js:390:1)
```
May consider passing in the following as the value `'1970-01-01T00:00:00.000'`